### PR TITLE
tmz 2.2.0 base58 base91 base122

### PR DESCRIPTION
**Pull request by t-m-z**
ready for review

### CODE REVIEW

File: lib/widgets/main_menu/licenses.dart [hunk 1]
```diff
@@ -78,6 +78,10 @@ class LicensesState extends State<Licenses> {
           ['Urwigo Tools', '@Krevo (WherigoTools)', 'MIT License'],
           ['Vigenère Breaker', 'Jens Guballa (guballa.de)', null],
           ['Whitespace Interpreter', 'Adam Papenhausen', 'MIT License'],
+          ['Base58', 'Dark Launch\nhttps://www.darklaunch.com/base58-encode-and-decode-using-php-with-example-base58-encode-base58-decode.html', ''],
```
Comment (S-Man42) [inline lib/widgets/main_menu/licenses.dart:81]: Please remove the urls from here. It makes the licenses view less readable. Putting the urls to the code parts is enough, I believe.

Then you can do:

name1\n
name2

license1\n
license2

:)

File: lib/widgets/registry.dart [hunk 2]
```diff
@@ -1140,6 +1147,14 @@ initializeRegistry(BuildContext context) {
       'base',
       'base85',
     ]),
+    GCWTool(tool: Base91(), i18nPrefix: 'base_base91', searchKeys: [
+      'base',
+      'base122',
```
Comment (S-Man42) [inline lib/widgets/registry.dart:1152]: Wrong search string

File: lib/widgets/main_menu/licenses.dart (hunk 1 above)
Comment (t-m-z) [reply lib/widgets/main_menu/licenses.dart:81]: check and done

File: lib/widgets/registry.dart (hunk 2 above)
Comment (t-m-z) [reply lib/widgets/registry.dart:1152]: ups corrected

**Pull request merged**